cluster/examples/kubernetes/ceph/toolbox.yaml

```diff
@@ -17,43 +17,24 @@ spec:
     spec:
       dnsPolicy: ClusterFirstWithHostNet
```

Comment:
The toolbox pod spec is also defined in the integration tests in the `GetRookToolBox()` method of `ceph_manifests.go`. Please make the same updates there.